pkg/cmd/config.go

```diff
@@ -289,6 +297,8 @@ func createAndPrintManifest(printOptions *printOpts) error {
 	// Version
 	cfg.Set(yamled.Path{"versions", "kubernetes"}, printOptions.KubernetesVersion)
 
+	cfg.Set(yamled.Path{"kubeletConfiguration", "ContainerLogMaxSize"}, printOptions.ContainerLogMaxSize)
```

Comment:
Very tiny nit: I would put this at the end, so variables are grouped by priority/importance.

Reply:
done